benchmarks/zu-en/jw300-baseline/README.md

```diff
@@ -0,0 +1,70 @@
+## Data 
+JW300 : Zulu to English (English-Zulu Reverse Machine Translation)
+
+Author: Musa Ntuli
+
+
+## Model 
+Link to google drive folder with model(https://drive.google.com/drive/folders/10MSShnH8-V4ssOCpbEvFW4ZdVkop7iVd?usp=sharing)
+
+
+## Model Architecture
+  ### Text Preprocessing
+    - Remove blank/empty rows : 9037(0.85 %) samples
+    - Removed duplicates from source text : 82999(7.88 %) samples
+    - Removed duplicates from target text : 5045(0.52 %) samples
+    - Removed all numeric-only text : 182(0.02 %) samples
+    - Removed rows where text is fewer than orequal to 8 characters long from source text: 6272(0.65 %) samples
+    - Removed rows where text is fewer than orequal to 8 characters long from target text: 713(0.07 %) samples
+    - Removed rows where text is in test set: 1068(0.11 %) samples
+    
+   ### BPE Tokenization
+    - vocab size : 4000 (superior results than 10X)
+    
+   ### Model Config
+    - Details in supplied config file but used fewer transformer layers than in default notebook, with more attention heads and lower embedding size
+    - Trained for 235000 steps
+    - Took few hours on a single P100 GPU on Google colab over a three days (stopped training  saved best model then reloaded that model the next day)
+
+## Analysis
+
+Example #0
+Source:     20 Ungaqonda ukuthi kungani kunjalo ngabaningi .
+Reference:  20 You can understand why that is the case with many .
+Hypothesis: 20 You can understand why so many .
+
+Example #1
+Source:     OFakazi BakaJehova endaweni yakini bayokujabulela ukuxoxa nawe okwengeziwe mayelana nemfundo yeBhayibheli eqhutshwa emphakathini wakini . *
+Reference:  Jehovah ’ s Witnesses in your area will be glad to share more information with you about the Bible education program that is currently being carried on in your community . *
+Hypothesis: Jehovah ’ s Witnesses in your area will enjoy discussing more about Bible education that are conducted in your society . *
+
+Example #2
+Source:     Ukuhlakanipha nokuqonda nako kwakubalulekile .
+Reference:  Wisdom and discretion were also important .
+Hypothesis: Wisdom and discernment was important .
+
+Example #3
+Source:     Ngemva komhlangano wezizwe eRome ngalelo hlobo , ngaba nelungelo lokuba khona emhlanganweni owawuseNuremberg , eJalimane .
+Reference:  After the international convention in Rome that summer , I was privileged to attend the convention in Nuremberg , Germany .
+Hypothesis: After the international convention in Rome that summer , I had the privilege of attending the convention in Nuremberg , Germany .
+
+    
+## Results
+  - BLEU dev: 28.48

```

Comment:
These are great results. Thank you for sharing also the insights into the data processing and the analysis.